src/pages/dataElementGroups/form/validate.ts

```diff
@@ -0,0 +1,27 @@
+import { setIn } from 'final-form'
+import { dataElementGroupSchema } from './dataElementGroupSchema'
+import type { FormValues } from './types'
+
+// @TODO: Figure out if there's a utility for this? I couldn't find one
+function segmentsToPath(segments: Array<string | number>) {
```

Comment:
Coul we extract out the one in `dataElements` to an util and use that?

Reply:
addressed